integration-tests/conftest.py

```diff
@@ -1,10 +1,27 @@
 import pytest
+import subprocess
 
 
+@pytest.fixture(scope="session")
+def install_katello_rpm(test_config):
+    if "satellite" in test_config.environment:
+        # install katello rpm before register system against Satellite
+        cmd = "rpm -qa 'katello*' | xargs --no-run-if-empty rpm -e"
```

Comment:
while `katello*` is good as pattern, is there a slightly more precise pattern? what about `katello-ca-consumer*`?